.circleci/config.yml

```diff
@@ -158,18 +169,39 @@ jobs:
           name: Install psql
           command: |
             sudo apt-get update -y && sudo apt-get install -y postgresql-client
+      - run:
+          name: Install mysql
+          command: |
+            sudo apt-get update -y && sudo apt-get install -y mysql-client
       - run:
           name: Wait for vertica
           command: |
             # Bail out trying after 30 seconds
             end=$((SECONDS+30))
             echo "Starting at second ${SECONDS:?} - ending at ${end:?}"
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker